README.md

```diff
@@ -1,6 +1,7 @@
 # data-covid19-sfbayarea
 
-Processes for sourcing data for the Stop COVID-19 SF Bay Area dashboard, which you can find [here](https://stop-covid19-sfbayarea.netlify.com/), or [on GitHub](https://github.com/sfbrigade/stop-covid19-sfbayarea).  
+Processes for sourcing data for the Stop COVID-19 SF Bay Area dashboard, which you can find [here](https://panda.baybrigades.org/), or [on GitHub](https://github.com/sfbrigade/stop-covid19-sfbayarea).
```

Comment:
While we’re fixing this, how about rewording things so the link is meaningful and more accessible text instead of “here?”

Maybe “…which you can find at [panda.baybrigades.org](https://panda.baybrigades.org), or…”

Or “Processes for sourcing data for the [Stop COVID-19 SF Bay Area dashboard](https://panda.baybrigades.org/). You can find the dashboard’s source code in the [sfbrigade/stop-covid19-sfbayarea project](https://github.com/sfbrigade/stop-covid19-sfbayarea) on GitHub.”

Probably plenty of other good ways to do it, too.

Reply:
That reads better, for sure. I'll make those additional changes.